package.json

```diff
@@ -86,7 +93,8 @@
     "sass-loader": "^13.3.2",
     "stylelint": "^14.0.0",
     "ts-jest": "^28.0.0",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "tsconfig-paths-webpack-plugin": "^4.0.1"
```

Comment:
Why we need this pkg? We don't have it in uikit

Reply:
It seems we don't have absolute imports in uikit

Reply:
I think we do not need them in components either

Reply:
Why? Is it clear for you from first glance what import like`../../../../../../something` means?

Reply:
U write a component, it should not know anything above itself (except some utils)

Reply:
as for me even for utils it's easier to read code/imports when they well structured

but it's okay, I'll remove this

Reply:
U have a ton of code in ur PR. It's hard to comprehend